modules/govuk_jenkins/manifests/jobs/bouncer_cdn.pp

```diff
@@ -17,22 +17,22 @@
   $api_key = undef,
   $service_id = undef,
   $app_domain = hiera('app_domain'),
+  $app_domain_internal = hiera('app_domain_internal'),
 ) {
-
-  $check_name = 'bouncer-cdn-configuration'
   $service_description = 'Configure Bouncer CDN service with transitioning sites'
-  $job_url = "https://deploy.${app_domain}/job/Bouncer_CDN/"
 
   file { '/etc/jenkins_jobs/jobs/bouncer_cdn.yaml':
     ensure  => present,
     content => template('govuk_jenkins/jobs/bouncer_cdn.yaml.erb'),
     notify  => Exec['jenkins_jobs_update'],
   }
 
+  $check_name = 'bouncer-cdn-configuration'
+  $job_url = "https://deploy.${app_domain_internal}/job/Bouncer_CDN/"
```

Comment:
I don't think this'll work, as the internal domain is for internal communication between apps, and isn't the same as the `production.govuk.digital` domain that you want here for Jenkins.

https://github.com/alphagov/govuk-puppet/blob/master/hieradata_aws/production.yaml#L3

Reply:
Oh yeah... I'm sure we've solved this before somewhere... let me try and find it.

Reply:
I've pushed up an amended commit based on https://github.com/alphagov/govuk-puppet/pull/8935.

Reply:
:+1: